study.md

```diff
@@ -43,16 +44,17 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+For an Array there is a predetermined size of the data structure, which can be returned instantaneously.
+
+For an enumerable there is no way to know what sort of structure is being operated on. Therefore in order to get the size of the enumerable it must go through and count as it goes. Does that mean 'count' on an infinite enumerable will count up forever?
```

Comment:
`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns each digit of π on an iteration has does not have a finite length.